packages/create/templates/hello-express/src/main.ts

```diff
@@ -0,0 +1,16 @@
+import { app } from './operations';
+import { DBOS } from '@dbos-inc/dbos-sdk';
+
+async function main() {
+  await DBOS.launch({expressApp: app});
+
+  const PORT = DBOS.runtimeConfig?.port ?? 9000;
+  const ENV = process.env.NODE_ENV || 'development';
+
+  app.listen(PORT, () => {
+    console.log(`🚀 Server is running on http://localhost:${PORT}`);
```

Comment:
Why do we need a separate `main.ts` and `operations.ts`?  If we use the new start command, can we combine these files?

Reply:
I left it as 2 partly so I could unit test.  There may be a different way of doing that... but keeping unit tests seemed like a good idea to me.